test/check.sh

```diff
@@ -652,6 +688,50 @@ it_can_check_with_tag_regex_with_cursor() {
   "
 }
 
+it_can_check_with_tag_regex_invert_true_with_cursor() {
+  local repo=$(init_repo)
+  local ref1=$(make_commit $repo)
+  local ref2=$(make_annotated_tag $repo "1.0-staging" "a tag")
+  local ref3=$(make_commit $repo)
+  local ref4=$(make_annotated_tag $repo "1.0-production" "another tag")
+  local ref5=$(make_commit $repo)
+  local ref6=$(make_annotated_tag $repo "2.0-staging" "tag 3")
+  local ref7=$(make_commit $repo)
+  local ref8=$(make_annotated_tag $repo "2.0-production" "tag 4")
+  local ref9=$(make_commit $repo)
+  local ref10=$(make_annotated_tag $repo "3.0-staging" "tag 5")
+  local ref11=$(make_commit $repo)
+  local ref12=$(make_annotated_tag $repo "3.0-production" "tag 6")
+  local ref13=$(make_commit $repo)
+
+  x=$(check_uri_with_tag_regex_invert_from $repo ".*-staging" true "2.0-staging")
+  check_uri_with_tag_regex_invert_from $repo ".*-staging" true "2.0-staging" | jq -e "
+    . == []
```

Comment:
I think this one here should return 2.0-production and 3.0-production

Reply:
https://bitbucket.org/product/guides/basics/four-starting-steps?utm_source=engagement-engine&utm_medium=email&utm_campaign=4-starting-steps_EML-6176&ep_click_type=primary

Reply:
@dij77: what are you trying to tell?

Reply:
@xtremerui are you sure? If this is true, maybe i need an explanation about the `cursor`.

Reply:
@xtremerui you are right. I try to fix this.